frontend/packages/kubevirt-plugin/integration-tests/v2v/vmware/v2v.vmware.import.negative.scenario.incorrect.names.ts

```diff
@@ -0,0 +1,76 @@
+import { asyncForEach, click } from '@console/shared/src/test-utils/utils';
```

Comment:
can we adhere to the naming convention in all of these? Meaning that they should end with `scenario.ts` name

Reply:
done